### PR TITLE
feat(rabbitmq-connector): Configurable concurrent

### PR DESCRIPTION
**Pull request by chenjr0719**

### CODE REVIEW

File: rabbitmq-http-connector/main.go [hunk 1]
```diff
@@ -44,7 +44,11 @@ func (conn rabbitMQConnector) consumeMessage() {
 	}
 
 	forever := make(chan bool)
-	sem := make(chan int, 10) // Process maximum 10 messages concurrently
+	concurrent = os.Getenv("CONCURRENT")
```
Comment (RealHarshThakur) [inline rabbitmq-http-connector/main.go:47]:
```suggestion
	concurrent := os.Getenv("CONCURRENT")
```

Comment (chenjr0719) [reply rabbitmq-http-connector/main.go:47]: Oops, sorry that I still not get used to golang syntax. 😅

File: rabbitmq-http-connector/main.go [hunk 2]
```diff
@@ -44,7 +44,11 @@ func (conn rabbitMQConnector) consumeMessage() {
 	}
 
 	forever := make(chan bool)
-	sem := make(chan int, 10) // Process maximum 10 messages concurrently
+	concurrent := os.Getenv("CONCURRENT")
```
Comment (RealHarshThakur) [inline rabbitmq-http-connector/main.go:47]:
```suggestion
        concurrent := 0
 	 if os.Getenv("CONCURRENT") != ""{
	 concurrent = os.Getenv("CONCURRENT")
	 }
```
You should probably do something like that. Thanks for contributing but can you please test out the PR once before pushing further changes?

Comment (chenjr0719) [reply rabbitmq-http-connector/main.go:47]: Sure, will do it at my local. And sorry to send a PR without testing. 🙇‍♂️

**Review by RealHarshThakur — APPROVED** [review]

**Comment by vishal-biyani** [comment]
Thanks for this one @chenjr0719 - looks great 🎉

**Pull request merged**